### PR TITLE
Update static copy of User Input screen.

### PR DESCRIPTION
**Pull request by asvinb**
## Summary

Addresses issue #2302 

## Relevant technical choices
Update static copy to the ones in the IB

## Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/js/components/user-input/UserInputQuestionWrapper.js [hunk 1]
```diff
@@ -104,12 +108,20 @@ export default function UserInputQuestionWrapper( props ) {
 	);
 }
 
+UserInputQuestionWrapper.defaultProps = {
+	backLabel: __( 'Back', 'google-site-kit' ),
+	nextLabel: __( 'Next', 'google-site-kit' ),
```
Comment (eugene-manuilov) [inline assets/js/components/user-input/UserInputQuestionWrapper.js:113]: @asvinb let's add it into markup as a fallback values ( `{ nextLabel || __( 'Next', 'google-site-kit' ) }` ) to make sure translations are loaded by the time we render those buttons.

File: assets/js/components/user-input/UserInputQuestionnaire.js [hunk 2]
```diff
@@ -153,8 +157,10 @@ export default function UserInputQuestionnaire() {
 					slug={ USER_INPUT_QUESTION_SEARCH_TERMS }
 					isActive={ activeSlug === USER_INPUT_QUESTION_SEARCH_TERMS }
 					questionNumber={ 5 }
-					title={ __( 'To help us identify opportunities for your site, enter the top three search terms that you’d like to show up for', 'google-site-kit' ) }
+					title={ __( 'To help us identify opportunities for your site, enter the top three search terms that best describe your site\'s content.', 'google-site-kit' ) }
```
Comment (eugene-manuilov) [inline assets/js/components/user-input/UserInputQuestionnaire.js:160]:
```suggestion
					title={ __( 'To help us identify opportunities for your site, enter the top three search terms that best describe your site’s content.', 'google-site-kit' ) }
```

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]
Everything looks good to me, just added two tiny comments that need to be addressed before I can approve it.

File: assets/js/components/user-input/UserInputQuestionWrapper.js (hunk 1 above)
Comment (asvinb) [reply assets/js/components/user-input/UserInputQuestionWrapper.js]: Updated @eugene-manuilov

File: assets/js/components/user-input/UserInputQuestionnaire.js (hunk 2 above)
Comment (asvinb) [reply assets/js/components/user-input/UserInputQuestionnaire.js]: Updated @eugene-manuilov

**Review by eugene-manuilov — APPROVED** [review]
Awesome! Thanks, @asvinb.

**Review by aaemnnosttv — APPROVED** [review]
Super!

**Pull request merged**